models/junos/security_policy/junos_security_policies.yml

```diff
@@ -0,0 +1,514 @@
+---
+GENERATOR_VERSION: '1.0'
+ANSIBLE_METADATA: |
+  {
+     'metadata_version': '1.1',
+     'status': ['preview'],
+     'supported_by': 'network'
+  }
+NETWORK_OS: junos
+RESOURCE: l3_interfaces
```

Comment:
RESOURCE: security_policies

Reply:
yes these were one of the many copy paste issues i noticed yesterday evening. I've fixed them :)